demo/src/HyperviewScreen.js

```diff
@@ -6,62 +6,59 @@
  *
  */
 
+import * as Constants from './constants';
 import React, { PureComponent } from 'react';
 import HandleBack from './HandleBack';
-import Hyperview from 'hyperview';
+import Hyperview from 'hyperview/src';
```

Comment:
is this change needed? look at apply only needed changes (without Prettier / renaming)